atlp-rwanda/atlp-pulse-fn
# ft:Upload Resume

# PR Description

- As a trainee, I want to be able to upload my resume into the system.

# Description of tasks that were expected to be completed

- Resume documents can be uploaded to the servers
- External links (e.g. to google drive) can also be added
- Coordinators and admin can view each trainee's resume

# How has this been tested?
- Clone the repo
- Checkout into `` ft-upload-resume ``  branch
- run  `` npm i ``
- Navigate to the local url on the browser
- Login as a normal user
- Click on profile icon on top right corner
- Click on Profile
- Click on Edit Profile
- Click on Upload resume, a panel containing two buttons will be displayed. 1. Upload from pc: This is for upload resume from your local pc, 2. Add external link: This is for adding resume from external link
- Submit the resume
- Login as admin or coordinator, 
- Click on Trainees in sidebar
- From three action icons in the trainee's table click the last icon to see trainee's details. 
- You can now see where to view the trainee's resume and download it if it is a document or go to the page if it is a link.


# Number of Commits
2
The number of commits should not exceed **2 commits**. In case they are more than that, please take your time and squash them.

# Screenshots (If appropriate)
![image](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/63926982/8a9cd647-9a15-41fa-adb7-7684832b0a83)

![image](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/63926982/438089d9-5148-4add-aeb8-77fa5f819509)

# Please check this Checklist before you submit your PR:
